src/fs/file.rs

```diff
@@ -259,44 +259,32 @@ impl<'dir> File<'dir> {
         self.metadata.file_type().is_socket()
     }
 
-    /// Determine the full path resolving all symbolic links on demand.
-    pub fn absolute_path(&self) -> Option<&PathBuf> {
-        self.absolute_path.get_or_init(|| std::fs::canonicalize(&self.path).ok()).as_ref()
+    /// Determine the absolute path without resolving symlinks.
+    pub fn absolute_path(&self) -> PathBuf {
+        static CWD: OnceLock<PathBuf> = OnceLock::new();
+        CWD.get_or_init(|| std::env::current_dir().unwrap()).with_file_name(&self.path)
```

Comment:
This is different than how canonicalize works above.  I'll have to think about the difference for a bit.  Will this work if the user enters `eza -l --mount ../../foo`?

Reply:
Indeed, now that I look at my code with fresh eyes, this look… more than dubious. It won’t work if `self.path` is in a subdirectory of the current directory, or when using `..` and `.`.

I need to use [`Path::join()`](https://doc.rust-lang.org/std/path/struct.Path.html#method.join) instead, it’ll normalize the path and it should work in all cases.